typings/index.d.ts

```diff
@@ -180,15 +181,15 @@ export class Giveaway<ExtraData = any> extends EventEmitter {
     readonly isDrop: boolean;
 
     private ensureEndTimeout(): void;
-    private checkWinnerEntry(user: User): Promise<boolean>;
-    public checkBonusEntries(user: User): Promise<number>;
+    private checkWinnerEntry(user: User): Awaitable<boolean>;
+    public checkBonusEntries(user: User): Awaitable<number>;
```

Comment:
I'm not the most theoretical guy but just some speculation on my part.
These 2 functions should be awaited/use promises.
But the "Awaitable" is defined with an "OR" non-promise value.
Wouldn't that maybe imply wrong usage capability of the function for the develeoper? _Might be wrongly worded by me_

/ the whole "promise | non-promise" is imo just for the values the user provides for the properties. "= no execution"
-> but the return values of the async functions should be treated as a promise because of "excecution with promises".

As said just thoughts of mine. / if my interpretation is just my coding "convenience" then i'm sorry

Reply:
yeah I missed that thank you.